### PR TITLE
Remove ttf files and extract dependencies from prawn 0.5.1

### PR DESCRIPTION
**Pull request by guicassolato**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 *.swp
 nbproject
 pkg
+.rvmrc
+.bundle
+Gemfile.lock
```
Comment (hallelujah) [inline .gitignore]: Why ignoring the Gemfile.lock?

File: prawn-core.gemspec [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  spec.platform = Gem::Platform::RUBY
+  spec.summary = 'A fast and nimble PDF generator for Ruby'
+
+  spec.authors = [
+    'Gregory Brown',
+    'Guilherme Cassolato'
+  ]
+  spec.email = [
+    'gregory.t.brown@gmail.com',
+    'guicassolato@gmail.com'
```
Comment (hallelujah) [inline prawn-core.gemspec]: It would be good to use your redhat email and maybe add the 3scale-system-dev one too

File: .gitignore (hunk 1 above)
Comment (guicassolato) [reply .gitignore]: It wasn't me. I came from an external commit that I cherry picked.

File: prawn-core.gemspec [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  spec.summary = 'A fast and nimble PDF generator for Ruby'
+
+  spec.authors = [
+    'Gregory Brown',
+    'Guilherme Cassolato'
+  ]
+  spec.email = [
+    'gregory.t.brown@gmail.com',
+    'gcassolato@redhat.com',
+    '3scale-system-dev@redhat.com'
```
Comment (hallelujah) [inline prawn-core.gemspec]: Oh wait sorry 😖 , this is a internal mailing list ... I dunno if we should do that.

**Comment by hallelujah** [comment]
Closing but keeping the branch

**Pull request closed without merging**